public/components/datasources/components/manage/data_connection.tsx

```diff
@@ -103,6 +108,99 @@ export const DataConnection = (props: any) => {
     },
   ];
 
+  const QueryOrAccelerateData = () => {
+    console.log(chrome);
+    switch (datasourceDetails.connector) {
+      case 'S3GLUE':
+        return (
+          <EuiFlexGroup>
+            <EuiFlexItem>
+              <EuiCard
+                icon={<EuiIcon size="xxl" type="discoverApp" />}
+                title={'Query data'}
+                description="Query your data in Data Explorer or Observability Logs."
+                onClick={() => application!.navigateToApp(observabilityLogsID)}
+              />
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiCard
+                icon={<EuiIcon size="xxl" type="bolt" />}
+                title={'Accelerate performance'}
+                description="Accelerate performance through OpenSearch indexing."
+                onClick={() => {}}
+              />
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiCard
+                icon={<EuiIcon size="xxl" type="database" />}
+                title={'Tables'}
+                description="Manually Define Tables"
+                onClick={() => application!.navigateToApp('opensearch-query-workbench')}
```

Comment:
Will keep it like this for now, but I think ideally there should be additional path needed for workbench to correctly display the right datasource?

Reply:
@sumukhswamy  Can help you out here. We need to pass in `current data source` here in URL.

Reply:
Will keep it general for now, and update it later when it is ready